docs/platform/tutorials/eventDetection/dataAcquisition.md

```diff
@@ -0,0 +1,283 @@
+
+Follow these steps:
+
+1. Install the `Quix Tracker` from the Google Play Store. 
+
+	[![Quix Play store image](https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png){width=200px}](https://play.google.com/store/apps/details?id=com.quix.quixtracker&gl=GB){target=_blank}
+
+2. Open the app and navigate to the `Settings` page via the menu
+
+3. Click the `SCAN QR CODE` button at the top of the settings page. Now continue to follow these steps. When directed, in step #17, you will scan the QR code using the Quix Tracker app
+
+4. In the Quix Portal, click the user icon in the top right of the browser
+
+	![Portal token menu](./tokenmenu.png){width=150px}
+
+5. Click `Tokens`
+
+6. Generate a token with any name and allow at least a few days before it expires
+
+7. Copy the token to your clipboard or somewhere safe
+
+8. In the Quix Library, search for `QR Settings Share`
+
+9. Click `Setup & deploy`
+
+10. Paste the token into the `token` field
+
+11. Click `Deploy`
+
+12. Open the QR Settings Share by clicking the "open in new window" icon
+
+	![QR Settings Share Service](./qrsettingsshareservice.png){width=300px}
+
+13. Enter a name and a device identifier into the input fields
+
+14. These can be any value such as your name and CAR-001
+
+15. Click `Generate Token`
+
+16. A QR code will be generated and presented to you in the UI
+
+	![QR Settings Share UI](./qrsettingshareui.png){width=600px}
+
+17. Scan the QR code using the Quix Tracker app
+
+	The app has now been configured with the access token, allowing it to communicate with Quix.
+
+18. Ensure the rest of the fields are configured as follows:
+
+	a. Topic field is set to `phone-data`
+	
+	b. `Notifications Topic` field is set to `phone-out`
+
+	c. `DeviceId` field is set to your chosen device identifier
+
+	d. `Rider` is set to your name
+
+	e. `Team` has a value e.g. `Quix`
+
+19. Select `Dashboard` from the menu
+
+20. Click the `START` button
+
+	This will open a connection to your Quix workspace and start streaming data from your phone.
+
+### Verify the live data
+
+Follow these steps to ensure that everything is working as expected:
+
+1. In the Quix Portal, navigate to `Data Explorer`
+
+2. Ensure you are on the `Live data` tab
+
+3. Under `Select a topic` select `phone-data`
+
+4. Under `select streams` select the active stream. This should be the only stream that exists
+
+5. Under `select parameters or events` select gForceX, gForceY and gForceZ
+
+6. The waveform view should display the live g-force data from your phone.
+
+7. Move or gently shake your phone and notice that the waveform reflects whatever movement your phone is experiencing
+
+!!! success
+	You have connected the Quix Tracker app to your workspace and verified the connection using the Live Data Explorer.
+
+## CSV data
+
+If you don’t have an Android device or you’d rather just stream some data we’ve provided then this is the data source for you! 
```

Comment:
> If you don’t have an Android device or you’d rather just stream some data we’ve provided then this is the data source for you! 

If you don’t have an Android device, or you’d rather stream some data provided for you, then use this data source.

Reply:
In the blog, absolutely. For dev docs I always have an eye on translation and/or developers whose first language is not English - keeping it simple and directive, and as easy as possible to read: You want this...do this. Just a suggestion though.